src/GraphQL/Types/Scalars/Enumeration/CaseInsensitiveEnumValues.cs

```diff
@@ -0,0 +1,44 @@
+using GraphQLParser;
+
+namespace GraphQL.Types;
+
+/// <summary>
+/// A class that represents a case insensitive set of enumeration definitions.
+/// </summary>
+public class CaseInsensitiveEnumValues : EnumValues
```

Comment:
Hmmm, so inheriting from `EnumValues` so that `FindByValue` does not need to be overridden.... but it duplicates the list and all the conversions to constant case ...